specification/servicefabric/resource-manager/readme.md

```diff
@@ -25,8 +25,10 @@ To see additional help and options, run:
 These are the global settings for the ServiceFabricClient API.
 
 ``` yaml
+title: ServiceFabricClient

```

Comment:
@juhacket @sergey-shandar 
This is a breaking change and redefine the Swagger that correctly says `ServiceFabricManagementClient`

Reply:
@juhacket could you send a PR to fix this problem ASAP. @lmazuel I missed this line which effects all versions of API.

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/1748